pkg/pillar/utils/waitfor.go

```diff
@@ -145,3 +146,29 @@ func handleOnboardStatusImpl(ctxArg interface{}, key string,
 	}
 	ctx.Initialized = true
 }
+
+// WaitForUserContainerd waits until user containerd started
+func WaitForUserContainerd(ps *pubsub.PubSub, log *base.LogObject, agentName string, warningTime, errorTime time.Duration) error {
+	stillRunning := time.NewTicker(25 * time.Second)
+	ps.StillRunning(agentName, warningTime, errorTime)
+	checkTicker := time.NewTicker(5 * time.Second)
+	initialized := false
+
+	for !initialized {
+		log.Functionf("Waiting for user containerd socket initialized")
```

Comment:
Probably makes sense for this to be log.Noticef so we can see if any microservice is stuck due to this.